README.md

```diff
@@ -324,8 +324,10 @@ eza’s options are almost, but not quite, entirely unlike `ls`’s.
 - **-r**, **--reverse**: reverse the sort order
 - **-s**, **--sort=(field)**: which field to sort by
 - **--group-directories-first**: list directories before other files
-- **-D**, **--only-dirs**: list only directories
-- **-f**, **--only-files**: list only files
+- **-D**, **--only-dirs**: list only directories and links to directories
+- **-f**, **--only-files**: list only files and links to files
+- **--only-links**: list only symbolic links
```

Comment:
Trying this branch I get `eza: Unknown argument --only-links`

Reply:
gonna investigate this

Reply:
This might be due to merge errors but fixed it